### DISCUSSION TITLE
posts/2023/11/17/01/

### DISCUSSION THREAD

**Discussion by giscus** [wcho21/omag#56] [Giscus]
# posts/2023/11/17/01/

논리학에서 컴퓨터까지 훑어보기

https://drop.rooi.dev/posts/2023/11/17/01/

**Comment by vagabond95** [comment]
어려운 내용을 쉽게 풀어내는 능력이 탁월하신 것 같아요. 재밌게 잘 읽고 갑니다!

**Reply by wcho21** [reply]
감사합니다.
